src/assignments/mars-rover.rst

```diff
@@ -0,0 +1,137 @@
+Assignment #3: Mars Rover
+=========================
+This task is going to put your unit tests, modules, and exceptions knowledge to use
+by writing tests and classes for the Mars rover named Curiosity.
+
+.. figure:: figures/curiosity-rover-selfie.jpg
```

Comment:
In terms of licensing, can we use this picture?

Reply:
Yes it's a NASA picture that is in the public domain.